migrations/20200921185342-create-share-link-for-old-observations.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  async up(db) {
+    var md5 = require('md5');
+
+    global.migrationMsg = "Created Shared Link For Old Observations"
+    
+    let observationDocuments = await db.collection('solutions').find({isReusable:false},{type:"observation"}).toArray();
```

Comment:
@priyanka-TL  Does this logic works ?? I think check should be {isReusable : false, type : observation}.

Reply:
@priyanka-TL  Also Only project _id and author as this 2 fields are only required for observation . Check https://docs.mongodb.com/manual/tutorial/project-fields-from-query-results/ for reference.

Reply:
Does this logic works ?? I think check should be {isReusable : false, type : observation}. its working properly

Reply:
@priyanka-TL  Yes check should be in that format.